### DISCUSSION TITLE
question: once I deploy a lua plugin to luarocks, so I can install it via this plugin manager?

### DISCUSSION THREAD

**Discussion by linrongbin16** [lumen-oss/rocks.nvim#25] [Q&A] [answered]
same with title

**Comment by mrcjkb** [comment, marked as answer]
Hey :wave:

Yes, this plugin manager can install plugins from LuaRocks.org.
~The UI is still a work in progress though :smile: 
For example, the installation script doesn't actually install this plugin yet.~

**Reply by linrongbin16** [reply]
the idea is really great, since it's using a true package management system luarocks, and handle real library dependencies.

but for now the installation dependency is quite big, I think only hard core users like me would try it.
